e3sm_to_cmip/runner.py

```diff
@@ -90,7 +90,7 @@ class CLIArguments:
     info_out: Optional[str]
 
     precheck: Optional[str]
-    logdir: Optional[str]
+    logdir: str
```

Comment:
`logdir` (CMOR log dir) will always have a value because the default is `cmor_logs` (via argparse)